org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -610,6 +610,41 @@ SsoApplicationManager:
         ]
       }
 
+SSoSynapseLlmDeveloper:
```

Comment:
looking again, maybe this shouldn't be specific to synapse?  it seems like this would better as a general purpose role for LLM devlopers?  Also the below `permissionSetName: 'llm-developer'` is pretty generic.  So maybe change this and the rest of the naming for it to `SsoLlmDeveloper`?

Reply:
Good point, we may have multiple accounts for LLM resources. Renaming.